### DISCUSSION TITLE
Add more `onErrorXXX` stream operators.

### DISCUSSION THREAD

**Discussion by He-Pin** [apache/pekko#927] [Ideas]
Motivation:
As a daily Java developer , I think the `onError*` operators are very easy to use.
eg: https://github.com/spring-projects/spring-framework/pull/31352
refs:https://github.com/reactor/reactor-core/issues/535

Currently:
|Reactor-core|Pekko|Suggestion|
|--|---|----|
|onErrorResume|recoverWith||
|onErrorComplete|onErrorComplete ( added https://github.com/apache/incubator-pekko/issues/624)|
|onErrorReturn|recover/recoverWith|
|onErrorContinue|withAttributes(...)|
|onErrorStop|withAttributes/defaultBehavior|
|onErrorMap|mapError|

**Comment by laglangyue** [comment]
if possile, can I work for the simpler one of them, but I am not similiar with this module.

**Comment by He-Pin** [comment]
@laglangyue Thanks, I would like to gather more information before anyone from the community put effects on it.
As a Java developer I think the current api is not that easy to use.

**Comment by laglangyue** [comment]
> As a Java developer I think the current api is not that easy to use.

fine，good luck

**Comment by mdedetrich** [comment]
I think that this should be discussed more generally on github discussions/mailing list.

I am not against this in principle and I also have my own ideas regarding the attributes boilerplate (i.e. why isn't there an implicit's api for attributes for Scala) but we really should look at this comprehensively before coding it.

**Reply by He-Pin** [reply]
I transfer to thread, yes , I have to say, mostly devs here are just CRUD boies:)
